src/main/java/org/tdl/vireo/controller/SubmissionController.java

```diff
@@ -990,9 +990,18 @@ public ApiResponse removeFile(@WeaverUser User user, @PathVariable Long submissi
             apiResponse = new ApiResponse(ERROR, "You are not allowed to delete license files!");
         } else {
             if (user.getRole().equals(Role.ROLE_ADMIN) || user.getRole().equals(Role.ROLE_MANAGER) || uri.contains(String.valueOf(hash))) {
-                JsonNode fileInfo = assetService.getAssetFileInfo(uri, submission);
+                String fileName = "";
+                String fileSize = "file not found";
```

Comment:
Assuming file not found is not necessary as the file should only be deleted and visible before submission complete. Administrators should be prevented from viewing the submission until it is complete unless assistance requested from the advisory office.